server/planning/assignments/assignments.py

```diff
@@ -1239,6 +1239,7 @@ def accept_assignment(self, assignment_id, assignee):
     # coverage details
     'planning': deepcopy(coverage_schema['planning']),
     'description_text': metadata_schema['description_text'],
+    'name': metadata_schema['name'],
```

Comment:
`name` isn't defined in the `metadata_schema` but actually in the `planning_schema`.

``` python
# 1. Import the planning_schema
from planning.planning.planning import planning_schema

# 2. Change metadata_schema to planning_schema
'name': planning_schema['name']
```